plugins/techdocs-module-addons-contrib/api-report.md

```diff
@@ -0,0 +1,33 @@
+## API Report File for "@backstage/plugin-techdocs-module-addons-contrib"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { ComponentType } from 'react';
+
+// @public
+export const ReportIssue: ComponentType<ReportIssueProps>;
```

Comment:
Wanna push back a little bit on the naming here, thinking about the global namespace of things being imported in the app. Perhaps there's some suffix that can be added to make it clear that it's a TechDocs addon?

Thinking stuff like `ReportIssueAddon`, `ReportIssueTechDocsAddon`, `TechDocsReportIssueAddon`

On a separate note I notice this ends up as a `CompontentType`, which is a bit unfortunate, since it adds the children prop even though that's not really supported. Might be worth switching the return type of `createTechDocsAddonExtension` to wrap `(props: ...) => JSX.Element` instead. Nothing that blocks this PR though